.readthedocs.yaml

```diff
@@ -4,6 +4,9 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.11"
+  jobs:
+    post_checkout:
+      - git fetch --unshallow || true
```

Comment:
What needs the full git clone?

Reply:
`reno` does. It traverses git history to pull version numbers.